### ISSUE TITLE
history.py <> ms_to_mph 

### ISSUE THREAD

**Issue by IanD42** [pimoroni/weatherhat-python#17]
Not sure but I think there is a typo in:-
![image](https://user-images.githubusercontent.com/33552425/198817069-acae07ad-8c65-4e31-8196-d2333b03f55d.png)
Should cms be ms

**Comment by cbeierlbuffs** [comment]
I agree IanD42. The "c" in "cms" should be removed

**Comment by bsimmo** [comment]
Bump, came across this error today as well.

There has been a pull for this for while it seems
https://github.com/pimoroni/weatherhat-python/pull/18

Could you merge the pull, I know you're all RP2040 boards now, but don't forget about us old fashioned HAT users ;-)

**Comment by Gadgetoid** [comment]
Merged! I need to tag a release and... release the code... though, which is rather more of an adventure since a lot of our stuff has bitrot rather horribly :cry: which you can see by the merged code exploding our CI/tests.

Hey I have a Weather HAT running 24/7 up on my office wall!

**Comment by bsimmo** [comment]
I know, I think you post updates on twitter about it?

I was just adding in my HomeAssistant autodetect mqtt stuff, or trying as learning as I do it, and thought I'd add history and send as mph, well it didn't like that. 😆 

You have too many products to look after now at Pimoroni  :-).
Let's see what Bookworm PiOS when released along with the newer Python3 does to all of this...

**Comment by Gadgetoid** [comment]
Just trying to unpick the packaging mayhem now! Good grief a lot changed in very little time and it's a monster of an update.

Yes I've been shouting about the weather HAT and graphs- just moved my datastore onto a "real" PC so I can throw all sorts of things at it with impunity.

Bookworm is going to rain on my parade. My nuclear option is to add the following to `~/bashrc` automatically and pretend the whole venv problem doesn't exist-

```bash
PY_ENV_DIR=~/.python_venv
if [ ! -f $PY_ENV_DIR/bin/activate ]; then
  printf "Creating user Python environment, please wait...\n"
  python3 -m venv --system-site-packages $PY_ENV_DIR
fi
export VIRTUAL_ENV_DISABLE_PROMPT=1
source $PY_ENV_DIR/bin/activate
```

**Comment by Gadgetoid** [comment]
Okay, I published version 0.0.2 yesterday evening. Hopefully it rounds up and fixes some of the bugs that have been floating here for a while.

**Comment by helgibbons** [comment]
Think the issues here should now be resolved - if not, please let us know!

@bsimmo The new release should work with Bookworm/Pi 5, if that's been something you've been waiting for :)
